workflows/VGP-assembly-v2/kmer-profiling-hifi-trio-VGP2/kmer-profiling-hifi-trio-VGP2.ga

```diff
@@ -154,17 +158,105 @@
         },
         "5": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/pick_value/pick_value/0.2.0",
             "errors": null,
             "id": 5,
             "input_connections": {
-                "operation_type|input_reads": {
+                "style_cond|type_cond|pick_from_0|value": {
+                    "id": 3,
+                    "output_name": "output"
+                }
+            },
+            "inputs": [],
+            "label": null,
+            "name": "Pick parameter value",
+            "outputs": [
+                {
+                    "name": "integer_param",
+                    "type": "expression.json"
+                }
+            ],
+            "position": {
+                "left": 271.5784037973972,
+                "top": 632.6981223972093
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/pick_value/pick_value/0.2.0",
```

Comment:
this looks like more than just a update ? Can you describe those changes, or was this maybe not intentional ?

Reply:
yes I also fixed invalid connections to set the default value of some parameters. I added it to the CHANGELOGS.